.pre-commit-config.yaml

```
@@ -0,0 +1,56 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.1.0
+  hooks:
+  - id: check-added-large-files
+  - id: check-toml
+  - id: check-yaml
+  - id: end-of-file-fixer
```

```suggestion
  - id: end-of-file-fixer
  - id: fix-byte-order-marker
  - id: mixed-line-ending
  - id: requirements-txt-fixer
```
If we're using the end-of-file-fixer, there is no harm in fixing more issues in my opinion

---

Cool looks fine✅